packages/react-components/react-dialog/src/stories/DialogAlert.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Dialog, DialogTrigger, DialogSurface, DialogTitle, DialogBody, DialogActions } from '@fluentui/react-dialog';
+import { Button } from '@fluentui/react-components';
+import story from './DialogAlert.md';
+
+export const Alert = () => {
```

Comment:
```suggestion
export const AlertDialog = () => {
```

Let's be explicit here and conform with the `alertdialog` role